containers/challengesscreen.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Actions } from 'react-native-router-flux';
+
+import {
+  AppRegistry,
+  StyleSheet,
+  Text,
+  ListView,
+  View,
+  ScrollView,
+  Dimensions,
+  TouchableHighlight
+} from 'react-native';
+
+import Grid from '../components/grid.js'
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    marginTop: 70
+  }
+});
+
+class ChallengesScreen extends React.Component {
+
+  challengesList = [
```

Comment:
This mock is not the same as the agreed on the api documentation. Use api mocks, and if you're missing something ask to the api responsible to change it.